packages/@okta/vuepress-theme-prose/global-components/NutritionFacts.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="nutrition-facts" v-if="$page.frontmatter.nutritionItems">
+    <div
+      class="fact-item"
+      v-for="(nutrition, index) in $page.frontmatter.nutritionItems"
+      v-bind:key="index"
+    >
+      <div class="fact-item-title">{{ nutrition.title }}</div>
+      <div class="fact-bullet-list">
+        <ul>
+          <li
+            v-for="(bulletItem, index) in nutrition.bulletList"
+            v-bind:key="index"
+          >
+            {{ bulletItem }}
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+/*
+Inside index.md of the page you like put <NutritionFacts></NutritionFacts>
+And special data set for that component
+ex.
+nutritionItems:
+  - title: This will cover
+    bulletList:
+      - Line one
+      - Line two
+  - title: Looking for something else?
+    bulletList:
```

Comment:
Even if I have only one item, do I have to use a bullet item? Does it make sense to have one item that is in a bullet list?

Reply:
could you provide an example of the data format you talking about?

Reply:
Here is an example.  In the Figma mockup below, the first row has an entry with only one row and that row has no bullet point. 

![image](https://user-images.githubusercontent.com/78044107/123831846-9842dc00-d8b9-11eb-987c-0bc2634f426d.png)

Reply:
well spotted, now I see,  I will extend the component with special logic for this case

Reply:
https://60ddee956e3f68008eb03352--reverent-murdock-829d24.netlify.app/docs/release-notes/

Reply:
I was able to set up the nutrition facts on a page but need some guidance on how to set one nutrition fact with only one item with no bullet point.  The example you give in the component all show nutrition facts with multiple items with bullets. Can you give an example with a single item with no bullet point?

Reply:
There is now no bullet when there is only one item which is good and I've given my LGTM.  However, I have one small nit, although the design team might have a bigger issue with it. I noticed that the text is indenting even when there is no bullet. See the screenshot below of current behavior.

![image](https://user-images.githubusercontent.com/78044107/124650524-b5922000-de4e-11eb-8bef-6f6f6168ead6.png)

Below are comparison screenshots showing the current and expected behavior.

![image](https://user-images.githubusercontent.com/78044107/124651460-cdb66f00-de4f-11eb-8f07-53d280903652.png)

@albertchen-okta

Reply:
here screenshot from the latest commit
<img width="773" alt="nutrition" src="https://user-images.githubusercontent.com/71866734/125082996-84288700-e0d0-11eb-9ca6-9f56393cf579.png">